packages/fetch/src/headers.js

```diff
@@ -8,21 +8,19 @@ import {types} from 'util';
 import http from 'http';
 import { isIterable } from './utils/is.js'
 
-const validators = /** @type {{validateHeaderName?:(name:string) => any, validateHeaderValue?:(name:string, value:string) => any}} */
-(http)
+/** @type {{validateHeaderValue?:(name:string, value:string) => any}} */
+const validators = (http)
 
-const validateHeaderName = typeof validators.validateHeaderName === 'function' ?
-	validators.validateHeaderName :
```

Comment:
Do we know exactly what this used to do?  Any chance we start inadvertently rejecting headers that use to be allowed by this?

Reply:
The `http`-based `validateHeaderName` looks the same as ours. The only difference now is the `:`

https://github.com/nodejs/node/blob/main/lib/_http_common.js#L206-L214